client/components/Tracker/Form/Form.jsx

```diff
@@ -1,11 +1,21 @@
 import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 import styled from 'styled-components';
 
 const Form = () => {
   const [feeling, updateFeeling] = useState('');
   const [thoughts, updateThoughts] = useState('');
 
   const handleSubmit = (event) => {
+    //(user_id, thought, feeling_id, track_date)
```

Comment:
I usually prevent the default action first

Reply:
Updated! Thanks!